tests/utilities/UIViewController+HUBSimulateLayoutCycle.m

```diff
@@ -0,0 +1,14 @@
+#import "UIViewController+HUBSimulateLayoutCycle.h"
+
+@implementation UIViewController (HUBSimulateLayoutCycle)
+
+- (void)hub_simulateLayoutCycle
+{
+    [self loadView];
```

Comment:
How about:
```objc
self.view; // TODO: Replace with [self loadViewIfNeeded] when we’ve dropped iOS 8 😬
```

Since `-loadView` is documented as a “You should never call this method directly.”.